test/BUILD.gn

```diff
@@ -456,7 +456,10 @@ test("brave_unit_tests") {
       "//brave/browser/ipfs/ipfs_host_resolver_unittest.cc",
       "//brave/browser/net/ipfs_redirect_network_delegate_helper_unittest.cc",
     ]
-    deps += [ "//brave/components/ipfs" ]
+    deps += [
+      "//brave/browser/ipfs/test:brave_browser_ipfs_unittests",
```

Comment:
```
    sources += [
      "//brave/browser/ipfs/content_browser_client_helper_unittest.cc",
      "//brave/browser/ipfs/ipfs_host_resolver_unittest.cc",
      "//brave/browser/net/ipfs_redirect_network_delegate_helper_unittest.cc",
    ]
```
Sources listed above should be moved to sources in brave_browser_ipfs_unittests too.